### PR TITLE
ROKS v4.11: Add missing PodSecurityViolation alert

### PR DESCRIPTION
**Pull request by rtheis**

### CODE REVIEW

File: assets/cluster-bootstrap/podsecurity-alert.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+spec:
+  groups:
+  - name: pod-security-violation
+    rules:
+    - alert: PodSecurityViolation
+      annotations:
+        summary: One or more workloads users created in the cluster don't match their Pod Security profile
+        description: >-
+          A workload (pod, deployment, deamonset, ...) was created somewhere in the cluster but it
+          did not match the PodSecurity "{{ `$labels.policy_level` }}" profile defined by its namespace either via the cluster-wide
```
Comment (jmcmeek) [inline assets/cluster-bootstrap/podsecurity-alert.yaml:16]: Don't know if matters, but this has:
```
"{{ `$labels.policy_level` }}"
```
While the source file has:
```
"{{ $labels.policy_level }}"
```
i.e. differs in backquotes

Comment (rtheis) [reply assets/cluster-bootstrap/podsecurity-alert.yaml:16]: Right, that is required in order to generate the proper bindata.

Comment (jmcmeek) [reply assets/cluster-bootstrap/podsecurity-alert.yaml:16]: okay.  approved.

**Review by jmcmeek — APPROVED** [review]

**Review by jonesbr17 — APPROVED** [review]
/lgtm
/approved

**Pull request merged**